packages/neuron-wallet/package.json

```diff
@@ -64,6 +64,8 @@
     "lint-staged": "9.2.0",
     "neuron-ui": "0.17.0-alpha.5",
     "rimraf": "2.6.3",
-    "spectron": "7.0.0"
+    "spectron": "7.0.0",
+    "ts-transformer-imports": "^0.4.3",
```

Comment:
Do not use version range

Reply:
Which rule claims that? Without any param like `exact` lerna add creates this by default. So this is really not a human mistake.

Reply:
I'll merge and confirm this. If we do have an explicit rule for this we'll have another PR. There're other dependencies with `^` it worthy a separate change.

Reply:
I've encountered a package who broken the semver rules and made everything down.

Reply:
Will send another PR updating that. But I don't think this rule should be required.